recognition/run_tests.py

```diff
@@ -0,0 +1,8 @@
+import os
+import unittest
+
+os.environ["TEST"] = "true"
```

Comment:
i imagine that you can't use the boolean type right?

Reply:
Environment variables are only strings